server.lua

```diff
@@ -58,7 +67,10 @@ RegisterNetEvent('hud:server:RelieveStress', function(amount)
     end
     Player.Functions.SetMetaData('stress', newStress)
     TriggerClientEvent('hud:client:UpdateStress', src, newStress)
-    TriggerClientEvent('QBCore:Notify', src, Lang:t("notify.stress_removed"))
+    if not notified then
+        TriggerClientEvent('QBCore:Notify', src, Lang:t("notify.stress_removed"))
+        notification()
+    end
```

Comment:
If that is the case, then it's likely no one will receive the notification, which would create a lot of confusion. Should it be tracked in a table with `source` or at bare minimum on the client side instead?

Reply:
I figured that its relating to the src, so it would only effectively fire up on the src client, from my testing, it did seem to work ok even with more then 1 player online. But I do agree it would be way more efficient client sided. I'll look into it.

Reply:
Yes. It would be related to the source, buuuuuut you are setting a server sided variable, although local, it's still global to the script. So any other clients will also reference that `notified` variable because the server doesn't know if it's the same client calling home. 

If the notified variable was a list instead, something like (really bad pseudocode incoming) `notified = { [src1] = true, [src2] = nil, [src3] = true, [src4] = false }`

Then when src 1 or src 3 calls the server, it will not send a notification. But if src 4 or src 3 call, then it will send a notification and set the index for that particular source ID. Since a source should be unique throughout every server session, it theoretically should work OK.

Again, it's late and I might be completely wrong in my train of thought. But I think it's the better way to do it if it the state is stored server side (unless doing it client side :).